collections/hub/golden_path/settlement_management/mixed_settlement_model.json

```
@@ -3242,21 +3278,23 @@
           "scripts": {
             "preRequest": {
               "exec": [
-                "console.log(environment.TESTFSP2_SDK_TESTAPI_WS_URL + '/callbacks/{$request.body.quoteId}');",
+                "//console.log(environment.TESTFSP2_SDK_TESTAPI_WS_URL + '/callbacks/{$request.body.quoteId}');",
                 "",
                 "if (environment.ENABLE_WS_ASSERTIONS===true) {",
-                "  await websocket.connect(environment.TESTFSP2_SDK_TESTAPI_WS_URL + '/callbacks/{$request.body.quoteId}', 'testfsp2callbacksq')",
+                "  //await websocket.connect(environment.TESTFSP2_SDK_TESTAPI_WS_URL + '/callbacks/{$request.body.quoteId}', 'testfsp2callbacksq')",
                 "}"
               ]
             },
             "postRequest": {
               "exec": [
                 "if (environment.ENABLE_WS_ASSERTIONS===true) {",
-                "  environment.testfsp2callbacks = await websocket.getMessage('testfsp2callbacksq', environment.WS_ASSERTION_TIMEOUT)",
+                "  await custom.sleep(500)",
```

If we are going to add a `custom.sleep(XXX)` then we should parameterise the value, refer to the following suggestion with added comment:

```suggestion
                "  // NOTE: This has been commented out due to the WS issue https://github.com/mojaloop/project/issues/3164, with a work-around to call the HTTP equivalent API on the postRequest instead until it can be resolved.",
                "  const customSleep = Number.parseFloat(environment.SLEEP_MAX_WAIT_IN_MS) || 250",
                "  await custom.sleep(customSleep)",
```